backend/game_engine/common.py

```diff
@@ -211,5 +224,64 @@ def to_json(self):
         return json.dumps(self.to_dict())
 
 
-def log_message(message):
-    logging.info(message)
+def log_message(message, game_attributes):
+    #game_attributes is usually self
+    print("LOG_MESSAGE CALLED --------------------")
+    phone_numbers=[]
+    tiktok=""
+    team_id=""
+    tribe_id=""
+    game_id=""
+    player_id=""
+    if game_attributes:
+        if hasattr(game_attributes, 'recipient_phone_numbers'):
+            phone_numbers=game_attributes.recipient_phone_numbers
+        elif hasattr(game_attributes, 'phone_number'):
+            phone_numbers=game_attributes.phone_number
+        if hasattr(game_attributes, 'game_id'):
+            game_id=game_attributes.game_id
+        elif hasattr(game_attributes, '_game_id'):
+            game_id=game_attributes._game_id
+        if hasattr(game_attributes, 'tiktok'):
+            tiktok=game_attributes.tiktok
+        elif hasattr(game_attributes, '_tiktok'):
+            tiktok=game_attributes._tiktok
+        if hasattr(game_attributes, 'team_id'):
+            team_id=game_attributes.team_id
+        if hasattr(game_attributes, '_team_id'):
+            team_id=game_attributes._team_id
+        if hasattr(game_attributes, 'tribe_id'):
+            tribe_id=game_attributes.tribe_id
+        if hasattr(game_attributes, '_tribe_id'):
+            tribe_id=game_attributes._tribe_id
+
+    with push_scope() as scope:
+        if phone_numbers:
+            #sometimes phone_numbers is not a list
+            if (isinstance(phone_numbers, list)):
+                for index, num in enumerate(phone_numbers):
+                    scope.set_tag("phone_number"+str(index), num)
+                    print("phone_number"+str(index) +" " + num)
+            else:
+                scope.set_tag("phone_number", phone_numbers)
+                print("phone_number " + phone_numbers)
+
+        if game_id:
+            print("game_id "+game_id)
+            scope.set_tag("game_id", game_id)
+
+        if player_id:
+            print("player_id "+player_id)
+            scope.set_tag("player_id", player_id)
+
+        if tiktok:
+            print("tiktok "+tiktok)
+            scope.set_tag("tiktok_hash", tiktok)
+
+        if team_id:
+            print("team_id "+team_id)
+            scope.set_tag("team_id", team_id)
+
+        #UNCOMMENT THE BELOW LINE TO ENABLE SENTRY CALLS
```

Comment:
With the updated method prototype, the push_to_sentry argument (or equivalent) can be default false, and used to conditionally perform remote logging. Then we can uncomment this line and switch the default to True for production.

If push_to_sentry is False, then log_message should simply call logging.info(...) with the formatted message text.